installation.rst

```diff
@@ -846,91 +846,105 @@ targets from the ``builddir`` directory in the source tree:
 
 Linux container runtimes like {Singularity} cannot run natively on
 Windows or Mac because of basic incompatibilities with the host kernel.
-(Contrary to a popular misconception, MacOS does not run on a Linux
+(Contrary to a popular misconception, macOS does not run on a Linux
 kernel. It runs on a kernel called Darwin originally forked from BSD.)
 
-For this reason, the {Singularity} community maintains a set of Vagrant
-Boxes via `Vagrant Cloud <https://www.vagrantup.com/>`__, one of
-`Hashicorp's <https://www.hashicorp.com/#open-source-tools>`_ open
-source tools. The current versions can be found under the `sylabs
-<https://app.vagrantup.com/sylabs>`_ organization.
+To run {Singularity} on a Windows or macOS computer, a Linux virtual machine
+(VM) is required. There are various ways to configure a VM on both Windows and
+macOS. We recommend the Windows Subsystem for Linux (WSL2), and Lima.
 
 Windows
 =======
 
-Install the following programs:
+Recent builds of Windows 10, and all builds of Windows 11, include version 2 of
+the Windows Subsystem for Linux. WSL2 provides a Linux virtual machine that is
+tightly integrated with the Windows environment. The default Linux distribution
+used by WSL2 is Ubuntu. It is straightforward to install {Singularity} inside
+WSL2 Ubuntu, and use all of its features.
 
--  `Git for Windows <https://git-for-windows.github.io/>`_
--  `VirtualBox for Windows <https://www.virtualbox.org/wiki/Downloads>`_
--  `Vagrant for Windows <https://www.vagrantup.com/downloads.html>`_
--  `Vagrant Manager for Windows <http://vagrantmanager.com/downloads/>`_
-
-{Singularity} Vagrant Box
--------------------------
-
-Run Git Bash (Windows) or open a terminal (Mac) and create and enter a
-directory to be used with your Vagrant VM.
+Follow the `WSL2 installation instructions
+<https://docs.microsoft.com/en-us/windows/wsl/install>`__ to enable WSL2 with
+the default Ubuntu 22.04 environment. On Windows 11 and the most recent builds
+of Windows 10 this is as easy as opening an administrator command prompt or
+Powershell window and entering:
 
 .. code::
 
-   $ mkdir vm-singularity-ce && \
-       cd vm-singularity-ce
+  wsl --install
 
-If you have already created and used this folder for another VM, you
-will need to destroy the VM and delete the Vagrantfile.
+Follow the prompts. A restart is required, and when you open the 'Ubuntu' app
+for the first time you’ll be asked to set a username and password for the Linux
+environment.
+
+You can install SingularityCE from source, or from the Ubuntu packages at the
+GitHub releases page. To quickly install the 4.0.0 package use the following
+commands inside the WSL2 Ubuntu window:
 
 .. code::
 
-   $ vagrant destroy && \
-       rm Vagrantfile
+  $ wget https://github.com/sylabs/singularity/releases/download/v4.0.0/singularity-ce_4.0.0-jammy_amd64.deb
```

Comment:
This is currently 404 (as it should be, of course, since the release hasn't happened yet), but we should remember to double-check that the link works once the release is out.